README.md

```diff
@@ -110,485 +129,49 @@ Workarounds for some bugs we encountered :
-### Microsoft Enterprise SSO plug-in for Apple devices
-
-Microsoft has recently released a new plug-in that uses the newly announced Apple feature called [Enterprise Single Sign-On](https://developer.apple.com/documentation/authenticationservices). Microsoft Enterprise SSO plug-in for Apple devices offers the following benefits: 
-
-* Comes delivered in Microsoft Authenticator app automatically and can be enabled by any MDM.
-* Provides seamless SSO for Active Directory joined accounts across all applications that support Apple's Enterprise Single Sign-On feature.
-* COMING SOON: Provides seamless SSO across Safari browsers and applications on the device.
-
-MSAL 1.1.0 and above will use Microsoft Enterprise SSO plug-in automatically instead of the Microsoft Authenticator app when it is active on the device. To use Microsoft Enterprise SSO plug-in in your tenant, you need to enable it in your MDM profile. 
-
-See [more information](https://docs.microsoft.com/en-us/azure/active-directory/develop/apple-sso-plugin) about configuring Microsoft Enterprise SSO plug-in for your device [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/apple-sso-plugin)
-
-### Single Account Mode
-
-If your app needs to support just one signed-in user at a time, MSAL provides a simple way to read the signed in account. This API must be also used when you are building an application to run on devices that are configured as shared devices - meaning that a single corporate device is shared between multiple employees. Employees can sign in to their devices and access customer information quickly. When they are finished with their shift or task, they will be able to sign-out of all apps on the shared device.
-
-Here is a code snippet that shows how you can retrieve current account. You must call API every time when your app comes to foreground or before performing a sensitive operation to detect any signed-in account changes. 
-
-#### Swift
-
-```swift
-let msalParameters = MSALParameters()
-msalParameters.completionBlockQueue = DispatchQueue.main
-                
-application.getCurrentAccount(with: msalParameters, completionBlock: { (currentAccount, previousAccount, error) in
-            
-	// currentAccount is the currently signed in account
-	// previousAccount is the previously signed in account if any
-})
+1. Check out the repository using the following command.
 ```
-
-#### Objective-C
-
-```objective-c
-MSALParameters *parameters = [MSALParameters new];
-parameters.completionBlockQueue = dispatch_get_main_queue();
-        
-[application getCurrentAccountWithParameters:parameters
-                             completionBlock:^(MSALAccount * _Nullable account, MSALAccount * _Nullable previousAccount, NSError * _Nullable error)
-{
-	// currentAccount is the currently signed in account
-	// previousAccount is the previously signed in account if any
-}];
+git clone https://github.com/AzureAD/microsoft-authentication-library-for-objc.git --recursive
```

Comment:
This will clone the `dev` branch, which may be unstable -- we should instead direct customers to consume an official release.

Reply:
Think we should consider linking off to Apple's instructions

https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app#Add-a-package-dependency